src/test/java/nl/atos/ontwikkelstraat/FormControllerSteps.java

```diff
@@ -0,0 +1,146 @@
+
+public class FormControllerSteps {
+    private final WebDriver webdriver;
+    public FormData formData;
+
+    public FormControllerSteps(SharedDriver webdriver) {
+        this.webdriver = webdriver;
+    }
+
+    @Before
+    public void setUp() {
+        formData = new FormData("john", "banks", "frontstreet", 12, "9485KS");
+    }
+
+    @After
+    public void tearDown() {
+        formData = null;
+    }
+
+    @Given("^That the website is running and I navigated to it$")
+    public void thatTheWebsiteIsRunningAndINavigatedToIt() throws InterruptedException {
+        webdriver.manage().window().maximize();
+        webdriver.manage().timeouts().setScriptTimeout(10, TimeUnit.SECONDS);
+        if (System.getenv("TRAVIS").equals("true")) {
+            webdriver.get("localhost:4445");
+        } else webdriver.get("http://localhost:8080/");
+
+    }
+
+    @When("^I fill in my firstName with \"([^\"]*)\"$")
+    public void iFillInMyFirstNameWith(String firstname) {
+        formData.setName(firstname);
+        webdriver.findElement(By.id("name")).sendKeys(firstname);
+        assertEquals(firstname, webdriver.findElement(By.id("name")).getAttribute("value"));
+        //System.out.println(webdriver.findElement(By.id("name")).getAttribute("value"));
+    }
+
+
+    @And("^I fill in my lastName$")
+    public void iFillInMyLastName()  {
+        formData.setSurname("Johnson");
+        webdriver.findElement(By.id("surname")).sendKeys("Johnson");
+        assertEquals("Johnson", webdriver.findElement(By.id("surname")).getAttribute("value"));
+    }
+
+    @And("^I fill in my StreetName$")
+    public void iFillInMyStreetName()  {
+        formData.setStreetName("hoofdstraat");
+        webdriver.findElement(By.id("streetName")).sendKeys("hoofdstraat");
+        assertEquals("hoofdstraat", webdriver.findElement(By.id("streetName")).getAttribute("value"));
+    }
+
+    @And("^I fill in my HouseNumber$")
+    public void iFillInMyHouseNumber()  {
+        formData.setHouseNumber(134);
+        webdriver.findElement(By.id("houseNumber")).sendKeys("134");
+        assertEquals("134", webdriver.findElement(By.id("houseNumber")).getAttribute("value"));
+    }
+
+    @And("^I fill in my Zipcode$")
+    public void iFillInMyZipcode()  {
+        formData.setZipCode("9485SL");
+        webdriver.findElement(By.id("zipCode")).sendKeys("9485SL");
+        assertEquals("9485SL", webdriver.findElement(By.id("zipCode")).getAttribute("value"));
+    }
+
+    @And("^I hit the submit button$")
+    public void iHitTheSubmitButton()  {
+        JavascriptExecutor jse = (JavascriptExecutor) webdriver;
+        jse.executeScript("window.scrollBy(0,250)", "");
+        webdriver.findElement(By.id("testSubmit")).click();
+    }
+
+    @Then("^I should see my name and address data on the next page and it is also saved$")
+    public void i_should_see_my_name_and_address_data_on_the_next_page_and_it_is_also_saved()  {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2347079493/issues/source?bid=3138095&fileBranchId=3208141#l92)